Features

javitocor opened the pull request:
Finished BACKGROUNDS-AND-GRADIENTS. Solo project.

javitocor committed:
Added skeleton to index.html

javitocor committed:
Added navigation bar style

javitocor committed:
Positioned images in main section, added background image

javitocor committed:
Images after section added and positioned

javitocor committed:
Added style and positioned footer

javitocor committed:
Aside element positioned

javitocor committed:
Fixed errors after validator

javitocor committed:
Fixed lintenrs errors

javitocor committed:
Updated readme file

rOluochKe reviewed (changes requested):
Hello @javitocor 
### You have done really well so far, this is impressive, however, there are a few issues you need to work on for your project to be approved.

- Write a more descriptive pull request, [check this guide](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)
- In your CSS file, avoid using the keyword `!important`

### Once you make all these changes, resubmit for another review. I wish you all the best in the next submission.

javitocor commented:
In this project, I cloned the Apple website (https://web.archive.org/web/20140301004610/http://www.apple.com/).
I have used different background images and gradients, as well as positioned elements in different ways.
After the first code review, I deleted the keyword !important and added a full description in the pull request.

javitocor committed:
Deleted !important keyword

rOluochKe reviewed (approved):
Hello @javitocor 
### Good job. You have done well :clap:
### I approve of this project. You can merge and submit the project completed form.
### I wish you all the best in the next project.
- Happy Coding!
